configuration/external_parameters.rst

```diff
@@ -40,13 +40,41 @@ the following ``VirtualHost`` configuration:
         </Directory>
     </VirtualHost>
 
+For Nginx web servers, the environment variables can be set with `fastcgi_param`_
+directive. For example in the configuration file where the ``fastcgi_params``
```

Comment:
The `fastcgi_params` file includes other server and execution environment information defined with `fastcgi_param` such as:
DOCUMENT_ROOT, SERVER_ADDR etc. It is not essential to be on this place. More because of a convenience and according to other examples as well. The custom defined environment variables would be set also without this file in PHP later on but the PHP `$_SERVER` would not be filled with other info.